doc/source/cookbook.rst

```diff
@@ -1354,3 +1354,37 @@ of the data values:
        'weight': [100, 140, 180],
        'sex': ['Male', 'Female']})
    df
+
+Common Excel Operations
+-----------------------
+
+Fill Handle
+***********
+
+Create a series of numbers following a set pattern in a certain set of cells.
+
+.. ipython:: python
+
+    df = pd.DataFrame({'AAA': [1]*8, 'BBB': list(range(0,8))}); df
+
+    # Fill numbers with difference 4 starting from 1 
+    # in rows 2 to 5 in column AAA.
```

Comment:
I think it'd be better to have all the explanations in the text before the Python block, instead of comments.